Unity3D/Assets/RosSharp/Scripts/PoseStampedTransformManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using UnityEngine;
```

Comment:
What is the difference of PoseStampedTransformManager to the PoseTransformManager class below (except the name and that the header is missing)?
Please delete it.